app/javascript/app/components/navbar/navbar-component.jsx

```diff
@@ -1,15 +1,22 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
 import Nav from 'components/nav';
 import ToolsNav from 'components/tools-nav';
 
 import styles from './navbar-styles.scss';
 
+const FEATURE_DATA_EXPLORER = process.env.FEATURE_DATA_EXPLORER === 'true';
```

Comment:
maybe it's a bit overkill but... wdyt of moving it to `feature-data-explorer` file (we could keep it in `features` folder) and implement a function `isEnabled` ? this way we could standardize features enabling and if the logic grows (this feature is enabled if this && this && this are met) we'd have that in one place.

Reply:
I think is a good idea 👍 , but should it belong to its own PR?

Reply:
if it's a feature that appears in this PR, I think it can be done in this PR, wdyt?

Reply:
It could be but would be nice to also do the same with `FEATURE_DATA_SURVEY` and maybe those changes get hidden in this PR.

Reply:
ok then, it may be another PR for that 👍